src/packages/mikroorm/src/base-resolver/provider.ts

```diff
@@ -95,6 +96,26 @@ export class MikroBackendProvider<D extends BaseDataEntity, G extends GraphQLEnt
 
 	public readonly supportsInFilter = true;
 
+	// Default backend provider config
+	public readonly backendProviderConfig: BackendProviderConfig = {
+		filter: {
+			root: true,
+			parentByChild: true,
+			childByChild: true,
```

Comment:
so `childByChild` is to enable filtering on a child entity within the graphql query?

Reply:
Yea that's what is now supported by https://github.com/exogee-technology/graphweaver/pull/102 related entities filter. I'm open to better naming conventions.